src/renderer/components/Onboarding/Screens/Welcome/index.js

```diff
@@ -21,6 +25,7 @@ const WelcomeContainer: ThemedComponent<*> = styled.div`
   align-items: center;
   flex-direction: column;
   position: relative;
+  background: ${p => (p.dark ? "#1E2A3B" : p.theme.colors.palette.background.paper)};
```

Comment:
this should not happen. Why is not in our color palette ?

Reply:
You are right, moved the wave colors to the palette